code/datums/dna.dm

```diff
@@ -672,6 +672,8 @@ GLOBAL_LIST_INIT(total_uf_len_by_block, populate_total_uf_len_by_block())
 		dna.features["tail_monkey"] = SSaccessories.tails_list_monkey[deconstruct_block(get_uni_feature_block(features, DNA_MONKEY_TAIL_BLOCK), length(SSaccessories.tails_list_monkey))]
 	if(dna.features["ears"])
 		dna.features["ears"] = SSaccessories.ears_list[deconstruct_block(get_uni_feature_block(features, DNA_EARS_BLOCK), length(SSaccessories.ears_list))]
+		var/obj/item/organ/internal/ears/ears = get_organ_by_type(__IMPLIED_TYPE__)
+		ears?.imprint_ear(src)
```

Comment:
I haven't tested genetics yet but we don't have a super good API for reacting to certain DNA blocks changing